api/src/events.rs

```diff
@@ -86,6 +142,10 @@ impl Events {
 
         let resolver = self.context.move_resolver()?;
         let events = resolver.as_converter().try_into_events(&contract_events)?;
-        Response::new(self.ledger_info, &events)
+
+        match accept_type {
+            AcceptType::Json => Response::new(self.ledger_info, &events),
+            AcceptType::Bcs => Response::new_bcs(self.ledger_info, &events),
```

Comment:
This PR should be reverted. The goal here was that events should be in their native form and never have gone through the resolver/converter. That's where the performance hit comes.